sentinelhub/config.py

```diff
@@ -252,105 +271,67 @@ def reset(self, params: object = ...) -> None:
             ``'sh_base_url'``. By default, all parameters will be reset and default value is ``Ellipsis``.
         :type params: Ellipsis or list(str) or str
```

Comment:
I see that we forgot to remove this `:type params:` line here. I'm also wondering if instead of `params: object = ...` we can set a type `params: Union[..., str, Sequence[str]] = ...`. Although I remember you were once mentioning there are some special issues with ellipsis typing :thinking:.

Reply:
yes, no Ellipsis type until 3.10 :broken_heart:

Reply:
Due to the informative nature of types I can do `Union[str, Sequence[str], object]` :thinking:

Reply:
went with the more informative type